WooCommerce/Classes/ViewRelated/Dashboard/Settings/Privacy/PrivacyBannerViewController.swift

```diff
@@ -0,0 +1,115 @@
+import Foundation
+import UIKit
+import SwiftUI
+import WordPressUI
+
+/// Hosting Controller for the Privacy Banner View
+///
+final class PrivacyBannerViewController: UIHostingController<PrivacyBanner> {
+
+    init(goToSettingsAction: @escaping (() -> ()), saveAction: @escaping (() -> ())) {
+        super.init(rootView: PrivacyBanner(goToSettingsAction: goToSettingsAction, saveAction: saveAction))
+    }
+
+    /// Needed for protocol conformance.
+    ///
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        // Needed when presented via the `BottomSheetViewController`
+        preferredContentSize = .init(width: 0, height: view.intrinsicContentSize.height)
+    }
+}
+
+extension PrivacyBannerViewController: DrawerPresentable {
+    var collapsedHeight: DrawerHeight {
+        return .intrinsicHeight
+    }
+
+    var expandedHeight: DrawerHeight {
+        return .intrinsicHeight
+    }
+}
+
+/// Banner View for the privacy settings.
+///
+struct PrivacyBanner: View {
+    /// Closure to be invoked when the go to settings button is pressed.
+    ///
+    let goToSettingsAction: (() -> ())
+
+    /// Closure to be invoked when the go to settings button is pressed.
```

Comment:
nit: comment needs to be updated for this closure

```suggestion
    /// Closure to be invoked when the save button is pressed.
```